### ISSUE TITLE
Remove warning about zipkin dependency in docs

### ISSUE THREAD

**Issue by codefromthecrypt** [micrometer-metrics/tracing#509]
This warning says brave has a dependency on zipkin, which is currently true for a transient reason. Once that goes away, the note here should be removed:

https://github.com/micrometer-metrics/tracing/blob/b8900270ea39d8e947a16b69d11489ec7acbad52/docs/modules/ROOT/pages/reporters.adoc?plain=1#L74

The reason is that brave temporarily pinned the zipkin reporter version in case the api drifted. No one removed that in 3 years, so it should be done shortly and once micrometer uses an updated version of brave, the above warning should go away: https://github.com/openzipkin/brave/blob/master/brave/pom.xml#L43-L49

Note: those who aren't using wavefront and choosing to use zipkin-reporter-brave will still have a sneaky dep on zipkin2 until a major version release of zipkin-reporter which would remove these deps. This would only affect those adding the zipkin reporter dependency so is a different thing than the wavefront thing, rather just a note fyi https://github.com/openzipkin/zipkin-reporter-java/issues/182

**Comment by codefromthecrypt** [comment]
This will be fixed in Brave 6, as most of the zipkin dependency was to help transition from functions deprecated 2 or more years ago.

**Comment by codefromthecrypt** [comment]
step one done https://github.com/openzipkin/zipkin-reporter-java/releases/tag/3.0.0

**Comment by codefromthecrypt** [comment]
note: otel is also pinning the zipkin types. I updated this, but they are still pinning them anyway. This doesn't affect the brave module directly, but as integration tests use a shared classpath between otel and brave, there are some glitches until merged and released https://github.com/open-telemetry/opentelemetry-java/pull/6129

**Comment by codefromthecrypt** [comment]
#511 will fix most of this except the actual doc change. The zipkin libs are still pulled in by otel and I added notes about what to do when https://github.com/open-telemetry/opentelemetry-java/pull/6129 is released

**Comment by codefromthecrypt** [comment]
specifically otel is locking into a version of zipkin-reporter-brave which still has a dep on zipkin types. It is easy to fix as I've raised the PR needed by OTEL and released library versions of what's needed to complete this.

The rest I have to leave with you all as I have to get to my day job which has nothing to do with tracing.

**Comment by codefromthecrypt** [comment]
finally I did a [brain dump](https://groups.google.com/g/zipkin-dev/c/x9xiqbVxrNk) here for those who care to parse it. 

TL;DR; zipkin server needs to be able to upgrade to a different floor version of java than instrumentation at some point, zipkin-reporter3  was needed not just for this issue, but also our own reasons

**Comment by AzazelSensei** [comment]
Opened https://github.com/micrometer-metrics/tracing/pull/1547
